worldedit-core/src/main/java/com/fastasyncworldedit/core/queue/implementation/QueueHandler.java

```diff
@@ -57,8 +57,7 @@ public abstract class QueueHandler implements Trimable, Runnable {
     private double targetTPS = 18;
 
     public QueueHandler() {
-        // TODO make main thread independent
-        // TaskManager.taskManager().repeat(this, 1);
+        TaskManager.taskManager().repeat(this, 1);
```

Comment:
The scheduling issues cannot be solved in this class.

Reply:
Its works but not great. When we solve the issue on the Task Manager. We can hold the the old code